showcase/src/assets/translations/nl.json

```diff
@@ -432,6 +432,49 @@
     "NEWS": {
       "TITLE": "Nieuws"
     },
+    "NGX_FORM_ERRORS": {
+      "TITLE": "Reactive Forms errors",
+      "EXAMPLE": "Voorbeeld",
+      "FORM": {
+        "VALIDATION": {
+          "REQUIRED": "{{fieldName}} is verplicht",
+          "PASSWORD_REQUIRED": "{{fieldName}} moet worden gegeven",
+          "USER_NAME": {
+            "UNIQUE": "Dit gebruikersnaam is al in gebruik"
+          },
+          "PASSWORD": {
+            "MAX_LENGTH": "Wachtwoord mag niet meer dan {{requiredLength}} tekens lang zijn",
+            "MIN_LENGTH": "Wachtwoord moet minimaal {{requiredLength}} tekens lang zijn",
+            "PATTERN": "Het wachtwoord moet minimaal één hoofdletter, één kleine letter en één cijfer bevatten"
```

Comment:
I would keep the sentence structure similar here. Eg: "Wachtwoord moet minimaal één hoofdletter, één kleine letter en één cijfer bevatten"